OJP/OJP_Trips.xsd

```diff
@@ -220,6 +258,23 @@
 			</xs:choice>
 		</xs:sequence>
 	</xs:complexType>
+	<xs:simpleType name="AccessibilityRoutingPreferenceEnumeration">
+		<xs:annotation>
+			<xs:documentation>Allowed values for a AccessibilityRoutingPreference.</xs:documentation>
+		</xs:annotation>
+		<xs:restriction base="xs:string">
+			<xs:enumeration value="levelEntranceOrBoardingAid"/>
+			<xs:enumeration value="noSingleStep"/>
+			<xs:enumeration value="noStairs"/>
```

Comment:
Just for my understanding: noSingleStep automatically also excludes noStairs?

Reply:
@JonaVBB It only excludes single steps, otherwise it should rather be called *noStep* - which would probably as well include *levelEntrance*

Reply:
It seems to me that we mean the same thing. Maybe my question was misleadingly formulated. I meant: if a user requests noSingleStep I understand that the user is not able to overcome a single step. So, I would furthermore assume that this user is then also not able to overcome stairs. Right?
I agree that noStep would include levelEntrance.

Reply:
@JonaVBB In the case of ***N**oSinlgeStep* as filter option (hard constraint) you may be right (though I'm not sure, it might be more precise to interpret it by just excluding the stated equipment, i.e., single steps).

In the case of the preferences (*noSingleStep* is from AccessibilityRoutingPreferenceEnumeration), my short answer would be: *noSingleStep* is only about single steps. The user does not *request* noSingleStep. He just prefers minimizing the number of single steps; minimizing stairs would have to be done separately (so as not to mix apples and pears when counting in order to minimize) by additionally specifying *noStairs*. In PR #273 I tried to give an example how to use the preferences. Do you have an idea whether & how to make this more clear for future users?

Reply:
@trurlurl OK, got it. Thank you.
Maybe it helps to emphazise in the documentation that noSingleStep and noStairs are technically independent from each other, although their wording might lead to interpretation that noSingleStep "includes" noStairs.
How do you distinguish between _minimizing_ steps/stairs and _totally avoiding_ them?